frontend/src/lib/components/PropertyFilters/components/PropertyValue.tsx

```diff
@@ -115,7 +115,7 @@ export function PropertyValue({
 
     const commonInputProps = {
         onSearch: (newInput: string) => {
-            setInput(newInput.trim())
+            setInput(newInput)
```

Comment:
Hmm, I don't think this is sufficient? You also want to prevent spaces at the end as well as the beginning. Those are usually bugs / non-intentional.

The current "hack" I use to get around this is to type `macos` and then add a space in the middle 😝, but would be very nice indeed to refactor into something that works for both cases

Reply:
I think the main problem is that we don't properly debounce all input. I had serious problems earlier with inputting some numbers as it was loading immediately when typing.